app/main.py

```diff
@@ -1,2 +1,48 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.load_factor = 5
+        self.filled_cells = 0
+        self.hash_table = [None] * self.capacity
+
+    def __setitem__(self,
+                    key: int | float | str | bool | tuple,
+                    value: Any) -> None:
+        index = hash(key) % self.capacity
```

Comment:
You should call resize() when size is equal to threshhold (threshhold is calculated as int(self.capacity * 2 / 3))